### DISCUSSION TITLE
Outpost security guards too easily killed and looted - breaks narrative logic

### DISCUSSION THREAD

**Discussion by Arctodites** [FakeFishGames/Barotrauma#14722] [Unstable feedback]
I have just reached the second outpost in my latest unstable playthrough, and both outposts have had outbursts of violence that resulted in two security guards being killed by hostiles.  That means I've been able to drag six bodies back to the sub and loot them for gear more valuable than my total mission earnings: four sets of body armor, weapons (smgs + clips, revolvers), helmets, 32 batteries . . . all without any repercussions.

What is more, these violent shootouts happen off-screen, and they lead to non-sensical event scenarios wherein all the key NPCs have either died or fled the scene by the time my player-character gets there to witness the supposed story event.

It is also bizarre to be able to drag the bloody corpses of multiple security guards through an outpost without anyone reacting, or without more guards arriving to at least stop and interrogate me.

![602960_14](https://github.com/user-attachments/assets/1f519043-58e7-4ab7-a97c-e318c2c02544)

**Comment by Regalis11** [comment]
The off-screen fights are actually a bug in the current unstable version, they are currently attacking all unconscious characters spawned by events. Closing as a duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14675

**Comment by Ek-Videogames** [comment]
Btw. Security guards can still die fairly easily due to 'intended' NPC fights - such as the separatist vandals, husk outbreaks (although to be fair husk are pretty scary so that sorta makes sense), and escaped specimens at science stations. Mainly because they will prioritize using their stun baton against all threats, which will quickly deplete it's battery after a couple of hits and be essentially worthless while they are in turn mauled or stabbed to death by their adversary.

Were guards too 'trigger happy' with SMGs in prior versions and were later changed? 
Or is this just an odd case of AI weapon prioritization?
